### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mhamidjamil**
#79 (#73 was working fine) #72
These issues were the main target of this PR.

### CODE REVIEW

File: v5/v5_managing_backup.ino [hunk 1]
```diff
@@ -1392,54 +1497,40 @@ void inputManager(String command, int inputFrom) {
...
     return true;
-  else
-    return false;
-}
-
-bool wapdaOn() {
-  if (digitalRead(WAPDA_STATE) == HIGH)
+  else if (mobileNumber.indexOf("Jazz") == -1 ||
+           senderNumber.indexOf("JAZZ") == -1 ||
+           senderNumber.indexOf("JAZZ OFFER") == -1 ||
```
Comment (wasey-rao) [inline v5/v5_managing_backup.ino:1525]: what is this ### senderNumber here, rejecting PR as this variable will cause error while compilation **fix it**

**Review by wasey-rao — CHANGES_REQUESTED** [review]
have a look at companyMsg function you have not initialize the **_senderNumber_**

File: v5/v5_managing_backup.ino (hunk 1 above)
Comment (mhamidjamil) [reply v5/v5_managing_backup.ino:1525]: yes! you are right, fixing it.

**Review by wasey-rao — APPROVED** [review]
Seems fine, and remember try to work in v5_stable code if this `managing_backup.ino` code is working now.

**Pull request merged**